test/cypress/e2e/register_challenge.spec.cy.js

```diff
@@ -442,6 +442,14 @@ describe('Register Challenge page', () => {
             .find('.q-radio')
             .first()
             .click();
+          cy.dataCy('step-4-info').should('be.visible');
```

Comment:
Could you test presence of div element with text _Have you noticed an incorrect or duplicate record? Please email us at [kontakt@dopracenakole.cz](mailto:kontakt@dopracenakole.cz)._ also, and add comment for these two cases, please?

Reply:
See updated test in `FormFieldSelectTable.cy`.
https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/303/files#diff-e28d5236d1e6eedadb0f92fb28d5168b81ccc4512956910c2ea03b792403bde3R90
I think the component is better suited for testing this as the e2e test is already very long.